config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -427,7 +427,7 @@ periodics:
         value: release-1.16
       - name: REPO_DIR
         value: /workspace/k8s.io/kubernetes
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20200731-2efbce0-1.16
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20200730-e636ee2-1.16
```

Comment:
I expect subsequent autobump PRs will try to change these back to whatever the latest built version is. If we still haven't found a fix-forward by then, we'll want to consider a manual tag